src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,446 @@
+| PharmaceuticalDoseRegimen | RECOMMENDED           | [string][]                             | Details of the pharmaceutical dose regimen. Either adequate description or short-code relating to regimen documented elsewhere (for example, `"single oral bolus"`).                                                                                                                                                                                                                                                               |
+| PharmaceuticalDoseTime    | RECOMMENDED           | [number][] or [array][] of [numbers][] | Time of administration of pharmaceutical dose, relative to time zero (please see below). For an infusion, this should be a vector with two elements specifying the start and end of the infusion period. For more complex dose regimens, the regimen description should be complete enough to enable unambiguous interpretation of the DoseTime vector. Unit format of the specified pharmaceutical dose time should be seconds. |
+| Anaesthesia               | OPTIONAL              | [string][]                             | Details of anaesthesia used, if any.                                                                                                                                                                                                                                                                                                                                                                                             |
+
+#### Radiochemistry
+
+This section is mandatory and contains information regarding the radioactive
+material used in the experiment.
+
+| **Key name**                  | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                       |
+| ----------------------------- | --------------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| InjectedRadioactivity         | REQUIRED              | [number][]    | Total amount of radioactivity injected into the patient (for example, 400). Corresponds to DICOM Tag (0018,1074) Radionuclide Total Dose.                                                                                                             |
```

Comment:
For bolus-infusion studies, two injected radioactivities should be reported. Therefore, we should consider to make the datatype for InjectedRadioactivity the following: [number][] or [array][] of [numbers][] with the bolus dose in the first position, and infusion in the second position (i.e. [bolus, infusion]). If either bolus or infusion, just a single number should be reported. The validator should be updated to require information about the infusion, if either a bolus-infusion or infusion scan is performed (i.e. ModeOfAdministration). Thoughts?  @mathesong @melanieganz

Reply:
Good catch! Another alternative might be to specify that the InjectedRadioactivity field is for the *total* injected radioactivity, and then to define a second InfusionInjectedRadioactivity, which specifies how much of the injected radioactivity came in the infusion (i.e. we can calculate bolus = total - infusion) for the case of a bolus+infusion paradigm. This ensures stability of the data type: we don't need to vary between a numeric or an array depending on the details of the measurement.

Reply:
I like this proposal. I will go ahead and suggest this change, including an additional item named InfusionStart to indicate the start time of the infusion.